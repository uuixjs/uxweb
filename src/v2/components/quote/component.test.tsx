import { Quote, QuoteProps } from ".";
import { setupShallowTest } from "../../tests/helpers";

function getRequiredProps(): QuoteProps {
  return {
    children: "Test",
  };
}

function getOptionalProps(): QuoteProps {
  return {
    padding: 1,
    children: "howdy",
    color: "#FF0000",
  };
}

const setupShallow = setupShallowTest(Quote);

describe("Quote", () => {
  it("renders quote component", () => {
    const { wrapper } = setupShallow(getRequiredProps());
    expect(wrapper).toMatchSnapshot();
  });

  it("renders quote component", () => {
    const { wrapper } = setupShallow(getOptionalProps());
    expect(wrapper).toMatchSnapshot();
  });
});
