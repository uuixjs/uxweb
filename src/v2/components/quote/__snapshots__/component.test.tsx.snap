// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders quote component 1`] = `
<Styled(Layout)
  className="tw-quote"
  padding={
    Object {
      "left": 0.5,
    }
  }
>
  Test
</Styled(Layout)>
`;

exports[`Quote renders quote component 2`] = `
<Styled(Layout)
  $color="#FF0000"
  className="tw-quote"
  padding={1}
>
  howdy
</Styled(Layout)>
`;
